validator/validate.py

```diff
@@ -31,7 +31,9 @@ def validate(path, format="json",
         The format to return the results in. Defaults to "json". Currently, any
         other format will simply return the error bundle.
     `approved_applications`:
-        Path to the list of approved application versions
+        Path to the list of approved application versions.
```

Comment:
Document that it can be a dict (same for the `library_metadata` below).

Should we remove the `app_versions.json` file from the repository now that it's not used anymore? That would be convenient, as we won't have to maintain it manually anymore